### PR TITLE
Enhancement/delete area

### PR DESCRIPTION
**Pull request by Mathematou**
Added the button to delete an area when creating or modifying a zone

### CODE REVIEW

File: app/src/main/java/com/github/sdpteam15/polyevents/model/map/ZoneAreaMapHelper.kt [hunk 1]
```diff
@@ -2,9 +2,13 @@ package com.github.sdpteam15.polyevents.model.map
 
 import android.content.Context
 import android.graphics.Color
+import android.util.Log
```
Comment (AlessioVerardo) [inline app/src/main/java/com/github/sdpteam15/polyevents/model/map/ZoneAreaMapHelper.kt:5]: Do we need this import ?

**Review by AlessioVerardo — APPROVED** [review]
So nice to have access to the delete zone button !

File: app/src/main/java/com/github/sdpteam15/polyevents/model/map/ZoneAreaMapHelper.kt [hunk 2]
```diff
@@ -139,6 +143,10 @@ object ZoneAreaMapHelper {
      * Clears the temporary variables to have a clean start for editing the area
      */
     fun createNewArea(context: Context?) {
+        if(deleteMode)
+            deleteMode(context)
+        if(editMode)
+            editMode(context)
         clearTemp()
         setupEditZone(context, GoogleMapHelper.map!!.cameraPosition!!.target)
     }
```
Comment (Cell00phane) [inline app/src/main/java/com/github/sdpteam15/polyevents/model/map/ZoneAreaMapHelper.kt:152]: Maybe add more comments to explain briefly what each mode means

**Review by Cell00phane — APPROVED** [review]
Nice improvement. Might be better with more comments. Otherwise all good!

**Pull request merged**